applications/app/controllers/MediaController.scala

```diff
@@ -52,7 +52,11 @@ class MediaController(contentApiClient: ContentApiClient, val controllerComponen
 
   private def renderMedia(model: MediaPage)(implicit request: RequestHeader): Result = {
     val htmlResponse = () => ContentHtmlPage.html(model)
-    val jsonResponse = () => views.html.fragments.mediaBody(model, displayCaption = false)
+    // The jsonResponse allows for a json version of each page to be accessed by users eg: https://www.thesguardian.com/world/2018/jun/13/kim-jong-un-north-korea-summit-trump-visit-kcna.json
```

Comment:
I added this comment because I found this jsonResponse code to be very confusing

Reply:
There is a tiny typo in "www.the_s_guardian.com"